### ISSUE TITLE
Service was not available at exposed port after machine's reboot

### ISSUE THREAD

**Issue by bzakdd** [moby/moby#35717]
**Description**

I've encountered following problem with docker swarm setup on 1 manager node:
I had multiple services. One of them, named service8 in the log, exposes port 80. I did following test in the loop:
- Reboot machine
- Wait until service8 becomes reachable under port 80

Reboots were performed at following hours:
- 15:51:00
- 16:04:38
- 16:09:49

After third reboot service8 became alive and healthy, but it was unreachable under port 80. There was a log repeating over and over again `miss notification: dest IP 10.255.0.4` where 10.255.0.4 was service8's ip address on docker's ingress network.
I waited for 30 minutes for it, then at 16:41:25 I did `docker service update --force service8` to fix the issue.

**Steps to reproduce the issue:**
This issue occured only once, I was unable to reproduce it.

**Describe the results you received:**
After reboot service was healthy, but unreachable under exposed port.

**Describe the results you expected:**
After reboot if service becomes healthy it is reachable under exposed port.

**Output of `docker version`:**

```
Client:
 Version:      17.06.2-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   cec0b72
 Built:        Tue Sep  5 19:59:06 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.06.2-ce
 API version:  1.30 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   cec0b72
 Built:        Tue Sep  5 20:00:25 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
Containers: 10
 Running: 9
 Paused: 0
 Stopped: 1
Images: 13
Server Version: 17.06.2-ce
Storage Driver: devicemapper
 Pool Name: vg00-docker--pool
 Pool Blocksize: 524.3kB
 Base Device Size: 10.74GB
 Backing Filesystem: xfs
 Data file: 
 Metadata file: 
 Data Space Used: 5.607GB
 Data Space Total: 21.47GB
 Data Space Available: 15.87GB
 Metadata Space Used: 1.704MB
 Metadata Space Total: 209.7MB
 Metadata Space Available: 208MB
 Thin Pool Minimum Free Space: 2.147GB
 Udev Sync Supported: true
 Deferred Removal Enabled: true
 Deferred Deletion Enabled: true
 Deferred Deleted Device Count: 0
 Library Version: 1.02.107-RHEL7 (2015-10-14)
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: active
 NodeID: k9gig7zfwv4vg24gilj4us312
 Is Manager: true
 ClusterID: 405lxrdoov57rlyo1fyqk4u02
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 2
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
  Force Rotate: 0
 Root Rotation In Progress: false
 Node Address: 10.9.4.178
 Manager Addresses:
  10.9.4.178:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 6e23458c129b551d5c9871e5174f6b1b7f6d1170
runc version: 810190ceaa507aa2727d7ae6f4790c76ec150bd2
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 3.10.0-327.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 3
Total Memory: 5.671GiB
Name: node-10-9-4-178
ID: EIGP:VRLO:45K2:VKCA:AP6D:GKTB:QVVB:V5A7:JK3O:PZFZ:JLEP:UJRJ
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: bridge-nf-call-ip6tables is disabled
```

**Logs:**
https://gist.github.com/bzak9LD/6367f77f739f864517b505f02402b098

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Swarm was setup on 1 manager node on VMs with CentOS 7.2 on KVM

**Comment by thaJeztah** [comment]
Let me close this ticket for now, as it looks like it went stale.
